public/less/variables.less

```diff
@@ -0,0 +1,10 @@
+/** Import base variables **/
+
+@import "../lib/bootstrap/less/variables.less";
+
+// Brand Colors
+@brand-primary: #0f245c;
```

Comment:
Got this one colour (RWAHS blue) from Morgan's CSS, the rest are placeholders for now.